application.go

```diff
@@ -86,6 +88,17 @@ func init() {
 
 	isGoTest = strings.Contains(strings.ToLower(os.Args[0]), "test")
 
+	if memcacheAddr, err = gz.ReadEnvVar("GZ_FUEL_MEMCACHED_ADDR"); err != nil && !isGoTest {
+		logger.Info("Missing GZ_FUEL_MEMCACHED_ADDR env variable. Memcached will not be available.")
+	}
+
+	globals.QueryCache = memcache.New(memcacheAddr)
```

Comment:
Have you tested what happens in the case this is not enabled? I read a bit of the ``memcache.New` code but couldn't determine this is going to work with an empty string.

Reply:
Yes, this works fine if it's not enabled.